### PR TITLE
Language tweaks

### PR DESCRIPTION
**Pull request by meganskye**
Update crab-home.md for clarity

### CODE REVIEW

**Comment by meganskye** [comment]
hope i did this right!

**Comment by HackFisher** [comment]
@wuminzhe Please help @meganskye resolve the conflicts, or close this pull request if you are not going to merge it.

File: docs/crab-airdrop.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 id: crab-airdrop
 title: Crab Network Airdrop
-sidebar_label: Airdrop
+sidebar_label: Airdrop 2020
 ---
-## Airdrop
+## Airdrop 2020
 
-A portion of the initial supply of the CRINGs have been airdropped to the holders of RING & DOT, they will get CRINGs for free.
+A portion of the initial supply of cRING will been airdropped to the holders of RING & DOT.
```
Comment (wuminzhe) [inline docs/crab-airdrop.md:8]: Darwinia Network
Crab Network
Subscan
RING 
KTON
CRING
CKTON

File: docs/crab-home.md [hunk 2]
```diff
@@ -5,22 +5,22 @@ sidebar_label: Crab Network
 slug: /
 ---
 
-Darwinia Crab Testnet (Crab for short) is a canary network with real economics for Darwinia, The positioning of Crab is similar to Polkadot's Kusama Network. Expect Chaos is a reasonable expectation.
+Darwinia Crab Testnet (Crab) is a canary network with real economic value for Darwinia, and its positioning is similar to Polkadot's Kusama Network. To Expect Chaos is a reasonable assumption!
```
Comment (wuminzhe) [inline docs/crab-home.md:8]: We are not calling `Crab Testnet`, but `Crab  Network`.

File: docs/crab-home.md [hunk 3]
```diff
@@ -5,22 +5,22 @@ sidebar_label: Crab Network
...
 
-Crab mainly provides simulation environment for Darwinia's upgrade and application deployment, perform various radical experiments. Crab includes not only the complete software operation environment and network environment, but also the economic environment.
+Crab mainly serves as a testbed and simulation environment for Darwinia network upgrades and application deployment on which radical experiments can be performed. It not only provides a complete network and software execution environment, but also an economic environment.
 
-Crab is the first cross-chain NFT blockchain in Polkadot ecology, providing smart contract solutions, compatible with the Ethereum virtual machine, and supporting Dapps such as Defi and NFT to easily migrate to Polkadot.
+Crab is the first crosschain Bridge in the Polkadot ecosystem with Ethereum Virtual Machine (EVM) compatibility that natively supports DeFi dApps and NFTs, allowing them to  migrate seamlessly crosschain to Polkadot.
```
Comment (wuminzhe) [inline docs/crab-home.md:12]: `crosschain` or `cross-chain`, which one is better? We are using `cross-chain` now.

**Comment by HackFisher** [comment]
@robinwei-itering please help resolve the conflicts, review & merge this PR.

**Comment by robinwei-itering** [comment]
@HackFisher I've resolved the conflicts locally, but I don't seem to have the permission to push to Origin.

**Pull request merged**